src/main/java/org/broadinstitute/consent/http/service/UserService.java

```diff
@@ -454,12 +454,135 @@ public void hasValidActiveERACredentials(User user) {
     }
   }
 
+  /**
+   * Compliance method defined in
+   * <a href="https://broadworkbench.atlassian.net/browse/DT-1607">this ticket</a> that implements
```

Comment:
I don't think it's a good idea to put links to JIRA in the source code. Instead, we require that all PRs have JIRA tickets and maintiners can easily see the blame for any code to find out the ticket that the code was added for. Putting it in the code results in duplicate tracking and adds more work for a developer.
```suggestion
   * Compliance method that implements
```

Reply:
This was specifically requested in the last PR to reference the truth table the code was built around (before the ticket criteria was changed).  @fboulnois  - mind if I remove the link?